cypress/page/upload.page.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable require-jsdoc */
+class UploadFilePage {
+    const text = 'terminal_input_screenshot.PNG';
+  private uploadUrlPage: string;
+  private browse: string;
+  private uploadbtn: string;
+  private titleFile: string;
+  private fileName: string;
+
+  constructor() {
+    this.uploadUrlPage = "demo.automationtesting.in/FileUpload.html";
+    this.browse = "#input-4";
+    this.uploadbtn = "button[title = 'Upload selected files']";
+      this.titleFile = ".file-footer-caption";
+    this.fileName = "terminal-input-screenshot.PNG";
+  }
+
+  public visitUploadPage(): void {
+    cy.visit(this.uploadUrlPage);
+  }
+  public uploadFile(): void {
```

Comment:
could you modify this method to receive the file to upload, in that way the page object will be enable to upload anyfile.